npm/cypress-schematic/src/builders/cypress/index.ts

```diff
@@ -18,7 +18,7 @@ Partial<CypressCommandLine.CypressOpenOptions>;
 
 type StartDevServerProps = {
   devServerTarget: string
-  watch: boolean
+  watch?: boolean
```

Comment:
Ah, yes I was confused because I thought these were the Dev Server Props - not the Props that cypress passing to the dev server ... 

Renamed it for more clearification. (prefixed with cypress)